### ISSUE TITLE
Types are not consistent across database engines

### ISSUE THREAD

**Issue by TheColorRed** [hasura/graphql-engine#9678]
### Version Information

Server Version:
CLI Version (for CLI related issue): v2.25.1-cloud.1

### Environment

If a user were to switch between databases `MySQL -> Postgres` and vise versa the queries would break. Also, I have a tool that builds hasura graphql queries, and having the types inconsistent across engines makes it hard to build queries between different types of databases.

### What is the current behaviour?

When constructing a query by primary key the type is an `integer` but in postgres, the type is an `Int`. 

MySQL:
```
query ($primary_0_0: integer!) {
  example_by_pk(plant_id: $primary_0_0) {
    example_column
  }
}
```

Postgres:
```
query ($primary_0_0: Int!) {
  example_by_pk(plant_id: $primary_0_0) {
    example_column
  }
}
```

### What is the expected behaviour?

The types should be the same across all databases engines.

**Comment by TheColorRed** [comment]
Looks like this has been fixed by a recent update.

**Comment by rsd1122** [comment]
@TheColorRed - yes this has been fixed in Hasura v2.26. Thanks for reporting it.
